app/views/articles/show.html.erb

```diff
@@ -31,6 +31,41 @@
     </script>
   <% end %>
 
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "mainEntityOfPage": "True",
+    "url": "<%= article_url(@article) %>",
+    "image": {
+      "@type": "ImageObject",
+      "url": "<%= article_social_image_url(@article) %>"
+    },
+    "publisher": {
+      "@context": "http://schema.org",
+      "@type": "Organization"
+    },
+    "logo": {
```

Comment:
Looks like logo should maybe be `within` publisher here?

https://developers.google.com/search/docs/data-types/article

Like this?

```
       "publisher": {
        "@type": "Organization",
        "name": "Google",
        "logo": {
          "@type": "ImageObject",
          "url": "https://google.com/logo.jpg"
        }
      },
```

Reply:
Great catch! I've made this change.